README.md

```diff
@@ -81,7 +81,7 @@ cd chapter
 
 Install dependencies:
 ```
-npm install
+npm install && cd client && npm install && cd ..
```

Comment:
Maybe we can split this command into separate lines:
```
npm install
cd client
npm install // client packages (someone can give a better desc here...)
```
No need for the `cd ..` I think.